great_expectations/validator/validator.py

```diff
@@ -689,12 +693,21 @@ def resolve_validation_graph(
                 validation_graph=graph, metrics=metrics
             )
 
+            # Check to see if the user has disabled progress bars
+            disable = len(graph.edges) < 3
```

Comment:
How did you arrive at this?

Reply:
This was the previous condition as set by Alex! I didn't want to remove old behavior so in the case we've enabled the progress bars, that original conditional still applies